ui/src/Components/notification/index.js

```diff
@@ -0,0 +1,28 @@
+import { TYPE_TO_COLOR } from "../../Constant/constant";
+import { useNotificationContext } from "../../Context/NotificationContext";
+
+const NotificationParent = () => {
+  const { notificationQue } = useNotificationContext();
+
+  return (
+    <div
+      data-testid="notification-parent"
+      className="flex justify-center items-center flex-col-reverse fixed top-4 w-max gap-2 transition-[height] duration-300 ease-[cubic-bezier(0.165, 0.84, 0.44, 1)]"
+      //this piece of code will increase or decrease the height of the wrapper accr to the no. of notifications present
+      style={{ height: `${notificationQue.length * 2.5}rem` }}
+    >
+      {notificationQue.map((alert, index) => (
+        <div
```

Comment:
Thinking about:

1. How we can make it accessible? There is no `HTML` tag for notification. But shall we use `section` over `div` in the parent container?
2. We need to add aria tags for the screen reader support:  `aria-live`. Read more here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/alert_role